src/util/is.ts

```diff
@@ -0,0 +1,13 @@
+import { GenericObject } from '../types';
+
+export const isArray = (value: unknown): boolean => Array.isArray(value);
+
+export const isObject = (value: unknown): boolean => Object.prototype.toString.call(value) === '[object Object]';
+
+export const isPlainObject = (value: unknown): boolean =>
+  isObject(value) &&
+  value.constructor === Object &&
+  !(value as GenericObject).nodeType &&
+  !(value as GenericObject).setInterval;
+
+export const isNotNull = (element: unknown): boolean => typeof element !== 'undefined' && element !== null;
```

Comment:
Instead of using a function to check for a negation, create a positive version:

```typescript
export const isNull = (element: unknown): boolean => typeof element === 'object' && !element;
```

You will prevent using `!isNotNull` which is weird (I know that code was there…). It's better to use `!isNull`.

Reply:
Added `isNull` as an additional function call.

Reply:
Now `isNotNull` is not necessary because you can just negate `isNull`.